app/src/ui/banners/windows-version-no-longer-supported-banner.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import { Octicon } from '../octicons'
+import * as OcticonSymbol from '../octicons/octicons.generated'
+import { Banner } from './banner'
+import { LinkButton } from '../lib/link-button'
+
+export class WindowsVersionNoLongerSupportedBanner extends React.Component<{
```

Comment:
Maybe we should also use this on the mac no longer supported versions too?

Reply:
Agreed! (But can wait to another PR)